website_base_multi_image/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_portal_supplier_base_multi_image,access_portal_supplier_base_multi_image,base_multi_image.model_base_multi_image_image,base.group_portal,1,1,1,0
```

Comment:
So any portal users can modify any image? :-1: 

write/create/unlink perms must be ir.rule-constrained, and I guess that should be made per submodule, so delete this please.